src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -609,6 +609,33 @@ public static boolean isEqualCollection(final Collection<?> a, final Collection<
         return isEqualCollection(collect(a, transformer), collect(b, transformer));
     }
 
+    /**
+     * Returns the hash code of Collection through the hash method of Equator.
+     * <p>
+     * If the input collection is null return 1.
```

Comment:
I think it returns `0`? Maybe something like: "When the collection is {@code null} it returns 0 (zero). The equator must not be {@code null}, otherwise a {@code NullPointerException} is raised." (sorry my rusty Javadoc, I think there's a way to format code/numbers/classes/etc).